components/LinksHomepage/index.js

```diff
@@ -45,9 +45,7 @@ export default function LinksHomepage() {
     <>
       <StyledWrapper>
         <StyledForm onSubmit={handleSubmit}>
-          <StyledTitle>
-            <p>Add Recipe</p>
-          </StyledTitle>
+          <StyledTitle>Add Recipe</StyledTitle>
           <label htmlFor="link">Link</label>
           <br />
           <input
```

Comment:
The `input` should have an `id` to refer to the `htmlFor` attribute of the label.